<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>771</width>
      <height>361</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Matricula</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nome</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Departamento</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Titulacao</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tipo de Contrato</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>789</x>
      <y>-10</y>
      <width>31</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>601</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Dados: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxSearch">
       <item>
        <property name="text">
         <string>Busca Sequencial - Nome</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Busca Sequencial - Matricula</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Busca Binaria - Nome</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Busca Binaria - Matricula</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="comboBoxOrder">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>411</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Matricula</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nome</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Departamento - Nome</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Titulacao - Nome</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tipo de Contrato - Nome</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Departamento - Titulacao - Nome</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Formas de Ordenacao:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>794</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
